Cargo.toml

```diff
@@ -15,10 +15,13 @@ travis-ci = {repository = "sile/rustracing_jaeger"}
 codecov = {repository = "sile/rustracing_jaeger"}
 
 [dependencies]
+crossbeam-channel = "0.3"
 hostname = "0.1"
 percent-encoding = "1.0"
 rand = "0.6"
-rustracing = "0.1.8"
+# TODO: Temporary for integration
+#rustracing = "0.1.8"
+rustracing = { git = "https://github.com/mixalturek/rustracing.git", branch="no-mutex" }
```

Comment:
I will merge this PR when this temporary line is fixed.

Reply:
Done.